### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by Ataullah128** [IBM/ai-privacy-toolkit#60] [Q&A]
AttributeError: CLASSIFIER_PROBABILITIES

**Comment by NataliRa** [comment]
Hi
please, describe more about your situation/code. When do you get this error? Have you tried the tests?
When you create a model wrapper, you need to provide not only the model itself but also its output_type: The type of output the model yields (e.g., for vector use ModelOutputType.CLASSIFIER_PROBABILITIES or ModelOutputType.CLASSIFIER_LOGITS, for value use ModelOutputType.CLASSIFIER_SCALAR or ModelOutputType.REGRESSOR_SCALAR)
